plugins/additional/additional.cpp

```diff
@@ -229,13 +229,21 @@ class RichTextCtrlComponent : public ComponentBase
 {
 public:
 	wxObject* Create(IObject* obj, wxObject* parent) override {
+
+		wxString text = obj->GetPropertyAsString(_("value"));
+
 		wxRichTextCtrl* richText = new wxRichTextCtrl( 	(wxWindow*)parent,
 															wxID_ANY,
-															wxEmptyString,
+															text,
 															obj->GetPropertyAsPoint(_("pos")),
 															obj->GetPropertyAsSize(_("size")),
 															obj->GetPropertyAsInteger(_("style")) | obj->GetPropertyAsInteger(_("window_style")));
 
+		if( !text.empty() )
```

Comment:
There must have been a slight misunderstanding here, i didn't mean to add padding inside the parentheses (like is seen in old code and shouldn't be used in new code) but just add a space after the if, so this should look like this:
```
if (!text.empty())
```

Reply:
Right, old style a bit confused me. Fixed.